### PR TITLE
Update revista-brasileira-de-ciencia-do-solo.csl

### PR DESCRIPTION
**Pull request by FSHigashikawa**
I updated the style to the new citation standards of journal

### CODE REVIEW

File: revista-brasileira-de-ciencia-do-solo.csl [hunk 1]
```diff
@@ -47,432 +47,217 @@
...
-        <text macro="issued-year"/>
-        <text prefix=", " macro="citation-locator"/>
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="issued"/>
       </group>
     </layout>
   </citation>
-  <bibliography hanging-indent="false" et-al-min="3" et-al-use-first="1" entry-spacing="1">
+  <bibliography et-al-min="0" et-al-use-first="0" entry-spacing="0" line-spacing="0">
```
Comment (rmzelle) [inline revista-brasileira-de-ciencia-do-solo.csl]: `et-al-min="0" et-al-use-first="0"`

This doesn't make any sense. This way no author names are printed.

**Comment by rmzelle** [comment]
Please always make sure your style validates. See http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2FFSHigashikawa%2Fstyles%2Fpatch-3%2Frevista-brasileira-de-ciencia-do-solo.csl&version=1.0.1 (for the current pull request), and https://github.com/citation-style-language/styles/wiki/Validation

**Comment by rmzelle** [comment]
Can you update the template link? It still points to ABNT (`<link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas" rel="template"/>`), but should probably point to one of the Vancouver styles.

**Comment by FSHigashikawa** [comment]
OK! Thank you!

**Comment by rmzelle** [comment]
Thanks!

**Comment by FSHigashikawa** [comment]
Hi @rmzelle, I have updated this style based on Vancouver-author-date, but the citation style not changed... When I install this style by Mendeley Desktop (Get More Styles) the citation is the same of previous form (before last update). However, when I put the .csl file manually in "citationStyles-1.0" folder of my computer,  the style is in according with my recent update... Why this happen?

**Comment by adam3smith** [comment]
Mendeley take 1-2 days to update styles.

**Comment by FSHigashikawa** [comment]
Hi @adam3smith, thank you!

**Comment by FSHigashikawa** [comment]
Mendely didn't update the style yet...

**Comment by cpina** [comment]
Styles should be updated every working day, automatically - I'll check what's happening.

**Comment by cpina** [comment]
I see an issue importing styles on our side, will fix here. Apologies.

**Comment by FSHigashikawa** [comment]
OK! Thank you very much!

**Comment by cpina** [comment]
By the way, you can go to View - Citation Styles - more styles and then paste:
https://raw.githubusercontent.com/citation-style-language/styles-distribution/master/revista-brasileira-de-ciencia-do-solo.csl

it will download the styles and use it - while we fix the issue.

Regards,

**Comment by FSHigashikawa** [comment]
OK!
Thank you very much!

Best regards,

**Comment by cpina** [comment]
And now is available in Mendleey Desktop - usual place, no work arounds.

**Pull request merged**